package.json

```diff
@@ -57,17 +57,17 @@
     "tap": "^14.0.0"
   },
   "scripts": {
-    "unit": "tap --100 --coverage --coverage-report=text ./tests/test_*.js",
-    "pretest": "npm run -s lint",
-    "test": "npm run -s unit",
-    "posttest": "npm run -s prettier:check",
-    "coverage": "tap --coverage-report=html && open coverage/lcov-report/index.html",
-    "mocha": "nyc mocha $(grep -lr '^\\s*it(' tests)",
-    "lint": "eslint \"**/*.js\"",
-    "prettier": "prettier --write \"**/*.@(js|json|md|ts|yml)\"",
-    "prettier:check": "prettier --check \"**/*.@(js|json|md|ts|yml)\"",
-    "dtslint": "dtslint types",
-    "semantic-release": "semantic-release"
```

Comment:
These changes will take some getting used to, but I like them!

One thing I noticed here is that it no longer lints when running `npm run test`. Should we re-add that? It seems like it could be useful for devs who are new to the project.

Reply:
I usually tend to run only what's expected in scripts. `test` run tests, `lint` runs linters etc. Maybe for newcomers it would be useful to have a pre-commit hook to automatically format and lint changed files? WDYT?

Reply:
We could also add `check` that runs `test`, `lint` and `format`.